build.sbt

```diff
@@ -380,6 +380,7 @@ lazy val circe = libraryProject("circe")
       circeCore,
       circeJawn,
       circeTesting % Test,
+      catsEffectLaws % Test
```

Comment:
I think we inherit this already.  The things we used to use out of it are gone, but I believe the dependency is still here.